instructor/compiler.py

```diff
@@ -347,17 +337,17 @@ def primary_anti_body(assignment):
         for band in a.bands.all():
             if western_blot.has_whole_cell_lysate:
```

Comment:
Is it possible to combine these if statements? Is it possible for `western_blot` to have more than one `has_` flag set to true at a time?